mobile/native/src/trade/order/orderbook_client.rs

```diff
@@ -34,8 +34,9 @@ impl OrderbookClient {
             let response = response.json().await?;
             Ok(response)
         } else {
-            tracing::error!("Could not create new order");
-            bail!("Could not create new order: {response:?}")
+            let error = response.text().await?;
+            tracing::error!(error, "Could not create new order");
```

Comment:
:wrench: Can we get rid of this log? It should be redundant with the error below.

Reply:
I thought the same, I just kept it because it was already there.